OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -22,6 +22,14 @@ class GivesExperienceInfo : ITraitInfo
 		[Desc("Grant experience for team-kills.")]
 		public readonly bool FriendlyFire = false;
 
+		[Desc("Modifier (in percent) with which the `Experience` value will be multiplied",
+			"before being granted to an actor.")]
+		public readonly int ActorExperienceModifier = 10000;
```

Comment:
Why do we inflate this by a factor of 100 (i.e. making it a percentage)? Wouldn't just `100` work as well?